features/admin.applications.v1/components/forms/inbound-oidc-form.tsx

```diff
@@ -1831,6 +2012,104 @@ export const InboundOIDCForm: FunctionComponent<InboundOIDCFormPropsInterface> =
+                showHybridFlowEnableConfig
+                && !isSystemApplication
+                && !isDefaultApplication
+                && !isM2MApplication
+                && (
+                    <>
+                        <Grid.Row columns={ 2 }>
+                            <Grid.Column mobile={ 16 } tablet={ 16 } computer={ 16 }>
+                                <Divider />
+                                <Divider hidden />
+                            </Grid.Column>
+                            <Grid.Column mobile={ 16 } tablet={ 16 } computer={ 16 }>
+                                <Box display="flex" alignItems="flex-end">
+                                    <Heading as="h4">
+                                        { t("applications:forms.inboundOIDC.sections" +
+                                        ".hybridFlow.heading") }
+                                    </Heading>
+
+                                    { applicationConfig.advancedConfigurations.showHybridFlowFeatureStatusChip && (
+                                        <div className="oxygen-chip-div" >
+                                            <Chip
+                                                label={ t("common:new").toUpperCase() }
+                                                className="oxygen-menu-item-chip oxygen-chip-new" />
+                                        </div>
+                                    ) }
+                                </Box>
+                                <Field
+                                    ref={ hybridFlowEnableConfig }
+                                    name={ HYBRID_FLOW_ENABLE_CONFIG }
+                                    required={ false }
+                                    children={
+                                        [
+                                            {
+                                                label: t("applications:forms.inboundOIDC.sections.hybridFlow.enable." +
+                                                "label"),
+                                                value: HYBRID_FLOW_ENABLE_CONFIG
+                                            }
+                                        ]
+                                    }
+                                    type="checkbox"
+                                    value = { initialValues?.hybridFlow?.enable? [ HYBRID_FLOW_ENABLE_CONFIG ] : [] }
+                                    listen={ hybridFlowConfigValuesChangeListener }
+                                    readOnly={ readOnly }
+                                    data-testid={ `${ testId }--hybridFlow-enable-checkbox` }
+                                />
+                            </Grid.Column>
+                        </Grid.Row>
+                    </>
+                )
+            }
+
+            {
+                showHybridFlowEnableConfig
+                &&
+                ( enableHybridFlowResponseTypeField || initialValues?.hybridFlow?.enable )
```

Comment:
Shall we keep these in a single line?

Reply:
Fixed with: https://github.com/wso2/identity-apps/pull/6271/commits/11295ba4d8273091b1934290f09ab0ab4d02ac0d